WordPress/gutenberg
# Block inserter: Improve alignment of block inserter search and close icons

## What?

Improve alignment of block inserter search and close icons.

Fixes: https://github.com/WordPress/gutenberg/issues/49756

## Why?

When using the block inserter search and clearing the search it's pretty much noticeable that the icons are not aligned and it creates a strange experience.


## Testing Instructions
1. Open a post or page.
2. Open the block inserter, and check the search icon position.
3. Type in your search, and confirm that the search and close icons are aligned.

## Screenshots or screencast 

https://user-images.githubusercontent.com/252415/236876439-855e19ab-fef7-41c7-b69b-437a7a00b459.mov
